python/tvm/contrib/cutlass/build.py

```diff
@@ -35,6 +35,8 @@ def has_cutlass():
```

Comment:
Add docstrings and comments to explain the purpose and usage of the code [medium]